lib/led-matrix/src/LEDMatrix.cpp

```diff
@@ -0,0 +1,78 @@
+#include "LEDMatrix.h"
+
+#include "mbed.h"
+
+LEDMatrix::LEDMatrix(PinName R, PinName G, PinName B)
+    : m_RChannel(R), m_GChannel(G), m_BChannel(B), m_continue_flashing(false) {
```

Comment:
You should explicitly initialize `m_flashing_red`, `m_flashing_blue`, `m_flashing_green` to false